parlai/mturk/tasks/turn_annotations/config/left_pane_text.html

```diff
@@ -0,0 +1,6 @@
+<br>
```

Comment:
It's assumed that people using this task *may* make their own files, but the files currently in the config folder will be loaded as the defaults if users don't specify their own files. This is why I haven't added them to .gitignore, because it's useful to have them around as defaults.

Ha, Mary originated the TODO_FIXMEs, I think - they're a good idea :)